vnext/src-win/IntegrationTests/WebSocketMultipleSendTest.js

```diff
@@ -0,0 +1,168 @@
+/**
+ * TODO: update copyright?
```

Comment:
>TODO: update copyright? [](http://example.com/codeflow?start=3&length=23)

Our code should have:
```
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License. 
```

Reply:
Though, we must not remove copyright set by another companies or people.
If all text in this file is written by you, then it must be OK to do.